src/ODE.jl

```diff
@@ -66,10 +66,10 @@ function hinit(F, x0, t0, tend, p, reltol, abstol)
     # Returns first step, direction of integration and F evaluated at t0
     tdir = sign(tend-t0)
     tdir==0 && error("Zero time span")
-    tau = max(reltol*norm(x0, Inf), abstol)
-    d0 = norm(x0, Inf)/tau
+    tau = max(reltol.*abs(x0), abstol)
```

Comment:
The "Hairer norm" used here isn't necessarily a norm because of how it it does the normalization.  But the tau here is not the standard sc since sc is the sum of the adjusted tolerances, not the max. In practice this shouldn't matter much though.